lib/pavilion/plugins/sched/slurm.py

```diff
@@ -45,9 +45,15 @@ def get_lines(self):
 
         lines = super().get_lines()
 
-        lines.append(
-            '#SBATCH --job-name "pav test #{s._test_id}"'
-            .format(s=self))
+         if self._conf.get('job_name') is not None:
+             lines.append(
+                 '#SBATCH --job-name {s._conf[job_name]}'
+                 .format(s=self))
+         else:
+             lines.append(
+                 '#SBATCH --job-name "pav test #{s._test_id}"'
+                 .format(s=self))
+
```

Comment:
The indentation is off here, which is breaking the whole plugin. Other than that this looks fine.

Reply:
fixed